test/cmocka/src/math/trig/cos_16B_fixed.c

```diff
@@ -0,0 +1,148 @@
+// SPDX-License-Identifier: BSD-3-Clause
```

Comment:
Nit about filename, B = byte, b = bit. We don't usually use capital letters in filenames, except when special attention is needed.